### ISSUE TITLE
Add support for milestones

### ISSUE THREAD

**Issue by minorOffense** [le1ca/gitlab-calendar#1]
Listing milestones in calendar would be great to see too. 

https://docs.gitlab.com/ee/api/milestones.html

**Comment by stratosgear** [comment]
I do not think a Gitlab webhook for Milestones exists, yet.

I wanted to implement something similar, and I hit the same wall.  See Gitlab [open issue](https://gitlab.com/gitlab-org/gitlab-foss/issues/13323) (Well, now forcibly closed) 

The API you mention, would only work if a cron-ish job was run every so often and compared the list of Milestones for any potential changes (quite hacky).

So sad... :(

**Comment by stratosgear** [comment]
Yes, but this only refers to the Milestone that the issue belongs to.

What happens when someone edits the end date for a Milestone.  Will this action trigger a new webhook so the milestone end date gets updated?  Haven't check lately the status ofthe Gitlab API, but I think not!

I hope you do not mean that you will implicitly "read" the state of a Milestone on every issue's webhook call.

**Comment by PeterDaveHello** [comment]
I don't think there's another trigger yet, unless somebody implemented it.
